cmds/tui.c

```diff
@@ -2215,8 +2215,23 @@ static void tui_main_loop(struct opts *opts, struct ftrace_file_handle *handle)
 			break;
 		case 'R':
 		case 'r':
+			if (win == &graph->win) {
+				struct tui_graph_node *curr = win->curr;
+
+				free(tui_search);
+				tui_search = xstrdup(curr->n.name);
+			}
```

Comment:
Maybe we could similar for graph.  'R' key is to go to the root of the report, and 'r' key is to go to the current node.

Reply:
I added 'R' key for doing that.